### PR TITLE
Update Japanese translations

### PR DESCRIPTION
**Pull request by tkusano**
* Updated as PR on Plural is merged (#3882)
* Feedback API (#3498)

etc.

### CODE REVIEW

File: src/locale/locales/ja/messages.po
Comment (dolciss) [inline src/locale/locales/ja/messages.po:26]: 英文の時点でplacedとappliedが混在しているのですが、ラベルが貼られているor適用されているで揺れているため統一できるとうれしいです。
```suggestion
msgstr "{0, plural, other {#個のラベルがこのアカウントに適用されています}}"
```

File: src/locale/locales/ja/messages.po
Comment (dolciss) [inline src/locale/locales/ja/messages.po:30]:
```suggestion
msgstr "{0, plural, other {#個のラベルがこのコンテンツに適用されています}}"
```
同様の箇所が以下の行にあります（"ラベル"でgrep）
貼る：546, 554, 5650
適用：3388, 4688, 4692, 4881, 4885,

**Review by dolciss — COMMENTED** [review]
Thank you for always submitting PRs for translations and for handling the plural form of #3382.
Please confirm my comment on the labeling notation shaking.

File: src/locale/locales/ja/messages.po
Comment (tkusano) [reply src/locale/locales/ja/messages.po]: すべて「適用」に統一してみました

**Comment by dolciss** [comment]
Thanks for applying your comment!
LGTM

**Comment by tkusano** [comment]
Updated translations related to following PRs
- #3934 
- #3937 
- #3913 
- #3921

File: src/locale/locales/ja/messages.po
Comment (tkusano) [inline src/locale/locales/ja/messages.po:75]: I'll leave this message untranslated until #3962  is resolved

**Comment by Hima-Zinn** [comment]
I will check this PR tomorrow.

File: src/locale/locales/ja/messages.po
Comment (middlingphys) [inline src/locale/locales/ja/messages.po:5015]:
```suggestion
msgstr "<0>{0}</0>によって適用されたラベルです。"
```

File: src/locale/locales/ja/messages.po
Comment (middlingphys) [inline src/locale/locales/ja/messages.po:5122]:
```suggestion
msgstr "会話を報告するには、会話の画面経由でメッセージを報告してください。モデレーターがあなたの報告の文脈を理解しやすくなります。"
```

**Comment by tkusano** [comment]
@middlingphys Fixed. thanks!

File: src/locale/locales/ja/messages.po
Comment (Hima-Zinn) [inline src/locale/locales/ja/messages.po:3142]:
```suggestion
msgstr ".jpgと.pngファイルのみに対応しています"
```

"い"が抜けている。

File: src/locale/locales/ja/messages.po
Comment (Hima-Zinn) [inline src/locale/locales/ja/messages.po:5028]:
```suggestion
msgstr "あなたによって適用されたラベルです"
```

上の２つと言い方をそろえる。
ここは最後に"."がないので、変更前が正しいようであればそれでもかまいません。

**Review by Hima-Zinn — COMMENTED** [review]
Please check.

**Comment by pfrazee** [comment]
Looks like there are some unresolved discussions here so Im going to hold off merging. Ping me when this is ready!

Thank you everybody! You rule!

**Comment by Hima-Zinn** [comment]
@tkusano 
Please check my review.
After that, ping pfrazee-san.

File: src/locale/locales/ja/messages.po
Comment (tkusano) [reply src/locale/locales/ja/messages.po]: 修正しました

File: src/locale/locales/ja/messages.po
Comment (tkusano) [reply src/locale/locales/ja/messages.po]: 揃えました

**Comment by tkusano** [comment]
@pfrazee It looks like it didn't make it to 1.82, could you please merge this? Also, I would appreciate it if you could merge #3962 as well, as there is a message that cannot be translated.

**Comment by haileyok** [comment]
@tkusano Got #3962 merged in for you 👍 thank all of you for this work!

**Comment by tkusano** [comment]
@haileyok thanks!

**Comment by tkusano** [comment]
Updated (cf. #4032 )

**Review by pfrazee — APPROVED** [review]
Thanks all!

**Pull request merged**
